docs/_data/table/cn/common-configuration.yml

```diff
@@ -22,6 +22,8 @@ alluxio.test.mode:
   设置在测试过程中是否启用某些特殊行为。
 alluxio.underfs.address:
   Alluxio在底层文件系统中的文件夹。
+alluxio.underfs.allow.set.owner.failure:
```

Comment:
For branch-1.3 let's default to the option which would give users consistent behavior as before. We can make this configurable in master.